linkedevents/handlers/place-handler.php

```diff
@@ -35,18 +35,10 @@ private function coordinateTransform($coordinates) {
       }
       
       public function updateResource($postId, $resource) {
-        if ($resource->getPosition() && $resource->getPosition()->getCoordinates()) {
-          $resource->getPosition()->setCoordinates(
-            $this->coordinateTransform($resource->getPosition()->getCoordinates()));
-        }
         $this->filterApi->placeUpdate($resource->getId(), $resource);
       }
       
       public function createResource($postId, $resource) {
-        if ($resource->getPosition() && $resource->getPosition()->getCoordinates()) {
-          $resource->getPosition()->setCoordinates(
-            $this->coordinateTransform($resource->getPosition()->getCoordinates()));
-        }
```

Comment:
After these changes the coordinateTransform is an unused method, so it should be removed